### PR TITLE
Updated animation timing, removed console logs

### PR DESCRIPTION
**Pull request by Omar-Shueb**

### CODE REVIEW

File: vr-wordle/src/Components/Notification.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import React from "react";
 import { Text } from "@react-three/drei";
 
 export default function Notification(props) {
-  console.log("notification rendering");
```
Comment (TamoorW) [inline vr-wordle/src/Components/Notification.js:5]: nice removal

File: vr-wordle/src/Components/Grid.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ function StaticLetter(props) {
 
   const springs = useSpring({
     color: props.submitted ? color : "#3a3a3c",
-    delay: 700 + props.index * 500,
+    delay: 600 + props.index * 600,
     config: { duration: 200 },
   });
```
Comment (tomw13) [inline vr-wordle/src/Components/Grid.js:30]: I like this, how did you come up with these numbers?

Comment (Omar-Shueb) [reply vr-wordle/src/Components/Grid.js:30]: just trial and error!

File: vr-wordle/src/Components/Grid.js [hunk 3]
```diff
@@ -45,8 +45,6 @@ function StaticLetter(props) {
       position={props.position}
       scale={1}
```
Comment (kiranaatkar) [inline vr-wordle/src/Components/Grid.js:46]: do you need to specify scale={1} or is this default?

File: vr-wordle/src/Components/Grid.js [hunk 4]
```diff
@@ -45,8 +45,6 @@ function StaticLetter(props) {
...
       onClick={() => {
-        console.log(active);
-        console.log("clicked");
```
Comment (tomw13) [inline vr-wordle/src/Components/Grid.js:49]: Thanks for removing these! Really annoying!!

File: vr-wordle/src/Components/Letter.js [hunk 5]
```diff
@@ -32,7 +32,7 @@ export default function Letter(props) {
       api.position.set(
         (x - size.width / 2) / aspect,
         -(y - size.height / 2) / aspect,
-        props.position[2]
+        1
```
Comment (tomw13) [inline vr-wordle/src/Components/Letter.js:35]: Love it!!

File: vr-wordle/src/Components/Table.js [hunk 6]
```diff
@@ -3,7 +3,6 @@ import { Box } from "@react-three/drei";
 import { useBox } from "@react-three/cannon";
 
 export default function Table({ position, rotation, args }) {
-  console.log("table rendering");
   const [ref] = useBox(() => ({
```
Comment (tomw13) [inline vr-wordle/src/Components/Table.js:6]: _Great_ work

File: vr-wordle/src/Components/Grid.js (hunk 3 above)
Comment (Omar-Shueb) [reply vr-wordle/src/Components/Grid.js:46]: I think you're right, that is the default, it should be removed
```suggestion
```

**Review by TamoorW — APPROVED** [review]

**Review by tomw13 — APPROVED** [review]

**Pull request merged**